packit/utils/extensions.py

```diff
@@ -0,0 +1,74 @@
+# MIT License
```

Comment:
> packit/utils/extensions.py

I am not sure about the name here. Can you please add a docstring for each file with the short description, please?

Reply:
added